README.adoc

```diff
@@ -273,7 +273,7 @@ If you want to start your app as quickly as possible (most people do) there are
 * Fix the location of the
 https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#boot-features-external-config-application-property-files[Spring Boot config file(s)]
 with `spring.config.location` (command line argument or System property etc.).
-* Switch off JMX - you probably don't need it in a container - with `spring.jmx.enabled=false`
+* Switch off JMX -- you probably don't need it in a container -- with `spring.jmx.enabled=false`
```

Comment:
Really, we should replace the em dashes with parentheses here. The formatting is right, though (two hyphens to make an em dash).